package ftp_model;

message DownloadReq
{
    optional bytes file_name = 1;
    optional int32 start = 2;
    optional int32 len = 3;
}

message DownloadResp
{
    optional int32 ret_code = 1;
    optional bytes ret_msg = 2;
    optional bytes content = 3;
    optional int32 total_file_size = 4;
}

message UploadReq
{
    optional bytes file_name = 1;
    optional int32 start = 2;
    optional int32 len = 3;
}

message UploadResp
{
    optional int32 ret_code = 1;
    optional bytes ret_msg = 2;
}
